src/main/java/org/moltimate/moltimatebackend/service/DockingService.java

```diff
@@ -290,16 +291,19 @@ public static MultipartFile fetchMacromolecule(String pdbID) {
 		}
 		
 		log.info("Fetching Macromolecule {} for Docking Request", pdbID);
-		
-		try {
+
+		RestTemplate template = new RestTemplate();
+
+		ResponseEntity<byte[]> file = template.getForEntity("https://files.rcsb.org/download/"+ pdbID + ".pdb", byte[].class);
```

Comment:
Might suggest refactoring this into a service, making it a retryable operation. Not overly important tho.